src/components/people.tsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import { useContext, useEffect } from 'react';
+import { getPeople } from '../api';
+import { MyContext } from './state';
+import { Loader } from './Loader';
+import { Error } from './error';
+import { Person } from './person';
+import { Person as PersonType } from '../types';
+
+export const People = () => {
+  const { person, setPerson, error, setError, setIsLoading, isLoading } =
```

Comment:
The variable 'person' is expected to be an array of people based on the context, but its name suggests it represents a single person. Consider renaming it to 'people' for better clarity.